config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
@AD7six What's the downside of using the filter and allow 'off' for being false? I only see the benefit of using the filter.

If you need to set '0' or '1' for debug, it's very confusing. Likewise, if it needs to be exactly 'true' and 'false', it's also confusing.